css/ckeditor.css

```diff
@@ -0,0 +1,4 @@
+.cke.cke_combopanel,
+.cke_combo__styles .cke_combo_text{
```

Comment:
I tried to find a way to adjust the width with JS. But for now I see no better way of adjusting the styles dropdown without putting much effort in it.